Fails to build for me:

```
[ 98%] Linking CXX executable CompositeTransformUtilTestDriver
[ 98%] Built target CompositeTransformUtilTestDriver
Scanning dependencies of target ANTSFetchData
[ 98%] Generating /tmp/nix-build-ANTs-2.2.0.drv-0/ANTs-37ad4e20be3a5ecd26c2e4e41b49e778a0246c3d-src/build/ExternalData/TestData/Data/r16slice.nii.gz
-- Fetching "file:////tmp/nix-build-ANTs-2.2.0.drv-0/ANTs-37ad4e20be3a5ecd26c2e4e41b49e778a0246c3d-src/.ExternalData/MD5/37aaa33029410941bf4affff0479fa18"
-- Fetching "http://slicer.kitware.com/midas3/api/rest?method=midas.bitstream.download&checksum=37aaa33029410941bf4affff0479fa18"
-- Fetching "http://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=37aaa33029410941bf4affff0479fa18&algorithm=MD5"
-- Fetching "http://www.itk.org/files/ExternalData/MD5/37aaa33029410941bf4affff0479fa18"
CMake Error at /tmp/nix-build-ANTs-2.2.0.drv-0/ANTs-37ad4e20be3a5ecd26c2e4e41b49e778a0246c3d-src/CMake/ExternalData.cmake:743 (message):
  Object MD5=37aaa33029410941bf4affff0479fa18 not found at:

    file:////tmp/nix-build-ANTs-2.2.0.drv-0/ANTs-37ad4e20be3a5ecd26c2e4e41b49e778a0246c3d-src/.ExternalData/MD5/37aaa33029410941bf4affff0479fa18 ("Couldn't read a file:// file")
    http://slicer.kitware.com/midas3/api/rest?method=midas.bitstream.download&checksum=37aaa33029410941bf4affff0479fa18 ("Couldn't resolve host name")
    http://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=37aaa33029410941bf4affff0479fa18&algorithm=MD5 ("Couldn't resolve host name")
    http://www.itk.org/files/ExternalData/MD5/37aaa33029410941bf4affff0479fa18 ("Couldn't resolve host name")
Call Stack (most recent call first):
  /tmp/nix-build-ANTs-2.2.0.drv-0/ANTs-37ad4e20be3a5ecd26c2e4e41b49e778a0246c3d-src/CMake/ExternalData.cmake:765 (_ExternalData_download_object)


make[2]: *** [Examples/TestSuite/CMakeFiles/ANTSFetchData.dir/build.make:68: ExternalData/TestData/Data/r16slice.nii.gz.md5-stamp] Error 1
make[1]: *** [CMakeFiles/Makefile2:9377: Examples/TestSuite/CMakeFiles/ANTSFetchData.dir/all] Error 2
make: *** [Makefile:163: all] Error 2
builder for ‘/nix/store/29vmlr8fp39l9z9w0cn6d2np5ikki2j7-ANTs-2.2.0.drv’ failed with exit code 2
error: build of ‘/nix/store/29vmlr8fp39l9z9w0cn6d2np5ikki2j7-ANTs-2.2.0.drv’ failed
```

--- a/pkgs/applications/science/biology/ants/default.nix
+++ b/pkgs/applications/science/biology/ants/default.nix
@@ -2,23 +2,26 @@
 
 stdenv.mkDerivation rec {
   _name    = "ANTs";
-  _version = "2.1.0";
+  _version = "2.2.0";
   name  = "${_name}-${_version}";
 
   src = fetchFromGitHub {
-    owner  = "stnava";
+    owner  = "ANTsX";
     repo   = "ANTs";
-    rev    = "4e02aa76621698e3513330dd9e863e22917e14b7";
-    sha256 = "0gyys1lf69bl3569cskxc8r5llwcr0dsyzvlby5skhfpsyw0dh8r";
+    rev    = "37ad4e20be3a5ecd26c2e4e41b49e778a0246c3d";
+    sha256 = "1hrdwv3m9xh3yf7l0rm2ggxc2xzckfb8srs88g485ibfszx7i03q";
   };
 
   nativeBuildInputs = [ cmake makeWrapper ];
   buildInputs = [ itk vtk ];
 
-  cmakeFlags = [ "-DANTS_SUPERBUILD=FALSE" "-DUSE_VTK=TRUE" ];
+  cmakeFlags = [ "-DANTS_SUPERBUILD=FALSE" "-DUSE_VTK=TRUE"
+                 # as cmake otherwise tries to download test data:
+                 "-DBUILD_TESTING=FALSE" ];
+
+  enableParallelBuilding = true;
 
   checkPhase = "ctest";
-  doCheck = false;
 
   postInstall = ''
     for file in $out/bin/*; do
@@ -27,7 +30,7 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    homepage = https://github.com/stnava/ANTs;
+    homepage = https://github.com/ANTxS/ANTs;
     description = "Advanced normalization toolkit for medical image registration and other processing";
     maintainers = with maintainers; [ bcdarwin ];
     platforms = platforms.unix;

